src/qml/AppLauncher.qml

```diff
@@ -34,7 +34,7 @@ import QtQuick.Controls.Styles.Nemo 1.0
 
 GridView {
     id: gridview
-    width: cellWidth * columns
```

Comment:
Why is this change needed?

Reply:
Removes unnecessary calculation

Reply:
But GridView doens't necessarily have same width as parent Page.